modular_skyrat/modules/vore/code/datums/components/vore.dm

```diff
@@ -0,0 +1,157 @@
+/datum/component/vore
+	var/list/obj/vbelly/bellies = list()
+	var/mob/living/owner
+	var/char_vars
+	var/vore_toggles = list()
+	var/selected_belly = 1
+	var/vore_enabled = TRUE
+	var/current_slot
+
+/datum/component/vore/Initialize()
```

Comment:
Add some code for blocking administrators from placing the vore component on stuff. Admins have a verb to put components on things.

Reply:
If admins do a shit job at using their own tools, they should be demoted. Coders aren't to blame for admins being brain damaged.